Dockerfile

```diff
@@ -1,18 +1,52 @@
-FROM ubuntu:trusty
+FROM ubuntu:bionic
 
 MAINTAINER Arjan Verkerk <arjan.verkerk@nelen-schuurmans.nl>
 
 # change the date to force rebuilding the whole image
 ENV REFRESHED_AT 1972-12-25
 
-# system dependencies
-RUN apt update \
-    && apt dist-upgrade --assume-yes \
-    && apt install --assume-yes \
-        python-pip \
-        python-scipy \
-    && apt-get clean --assume-yes \
-    && pip install setuptools --upgrade \
-    && pip install zc.buildout --upgrade
 
+# Get rid of debconf messages like "unable to initialize frontend: Dialog".
+# https://github.com/docker/docker/issues/4032#issuecomment-192327844
+ARG DEBIAN_FRONTEND=noninteractive
+
+
+# Note: The official Debian and Ubuntu images automatically run apt-get clean,
+# so explicit invocation is not required. See RUN apt-get in "Best practices
+# for writing Dockerfiles". https://docs.docker.com/engine/userguide/↵
+# eng-image/dockerfile_best-practices/
+RUN apt-get update && apt-get install -y \
+    locales \
+    python3 \
+    python3-pip \
+&& apt-get clean -y && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
According to the above note, `apt-get clean` is redundant?

Reply:
Oops, copy-paste error.